modules/OCPP/OCPP.hpp

```diff
@@ -91,6 +91,8 @@ class OCPP : public Everest::ModuleBase {
     void init();
     void ready();
 
+    ocpp1_6::ChargePointErrorCode getOcppErrorCode(json evse_error);
```

Comment:
this function here will be deleted, the next time the ev-cli touches this file. in auto generated files you're only allowed to add code in between these // ev@... blocks
but if you make this function static and local to the OCPP.cpp, you don't need to declare it here at all